patches/minecraft/net/minecraft/world/gen/ChunkGeneratorOverworld.java.patch

```diff
@@ -100,3 +113,51 @@
          BlockFalling.field_149832_M = false;
      }
  
+@@ -469,6 +505,7 @@
+             flag |= this.field_185980_B.func_175794_a(this.field_185995_n, this.field_185990_i, new ChunkPos(p_185933_2_, p_185933_3_));
+         }
+ 
++        flag |= this.structureManager.retroGenerateStructures(this.field_185995_n, this.field_185990_i, new ChunkPos(p_185933_2_, p_185933_3_), p_185933_1_);
```

Comment:
Should only occur if `mapFeaturesEnabled` is true

Reply:
Why? The other methods generate are called regardless of this value, so a mod might decide to generate something anyway (even though not recommended).